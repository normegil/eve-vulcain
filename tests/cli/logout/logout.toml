bin.name = "eve-vulcain"
args = "logout -vvv"
status.code = 0
