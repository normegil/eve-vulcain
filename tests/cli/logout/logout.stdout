Successfully logged out.
Call eve-vulcain login to restart login process.
